xbmc/windowing/Resolution.cpp

```diff
@@ -131,8 +131,7 @@ void CResolutionUtils::FindResolutionFromWhitelist(float fps, int width, int hei
 
     // allow resolutions that are exact and have the correct refresh rate
     // allow macroblock alignement / padding errors (e.g. 1080 mod16 == 8)
-    if (((height == info.iScreenHeight && width <= info.iScreenWidth + 8) ||
-         (width == info.iScreenWidth && height <= info.iScreenHeight + 8)) &&
+    if ((height == info.iScreenHeight && width == info.iScreenWidth + 8)) &&
```

Comment:
This case will barely be hit as with h264 the width (real width) must be mod 16. Most rips you get are either 1920x1088 if full screen or 1920x800 or 1920x816. The above does not make sense in this combination.

Reply:
Does not apply anymore :)